src/manual/products.md

```diff
@@ -12,16 +12,16 @@ The list view provides a paged list of all available products in the system, whi
 
 ## Default articles
 
-Articles (product type: "Default") are the base for almost all products that are added to the shop. Each article contains its own SKU and therefore its own stock level.
+An *Article* (product type / product code: "Default") is the base for almost all products that are added to the shop. An *article* contains its own SKU and therefore its own stock level. All *articles* must be assigned to at least one category in order to be visible, selectable and searchable in the frontend.
 
 ![Product article](Admin-product-article.png)
 
-Aimeos can offer discounts of various types when the consumer enters a coupon code. In case of monetary discounts, the amount is assigned to the configured (discount) article. In that case, the article is only used for giving the discount and it shouldn't be associated to a category. This ensures that the product is not added to the catalog index and therefore, not listed in any search results. It also prevents users from adding that product to their basket at all.
+There is something like a **non-categorizable article**, though. This is actually not a separate product type, but a *default article* product, which *must not* be added to any category, reason being that such an *article* is only used in conjunction with [vouchers](#vouchers) (not to be confused with a *voucher product*!) and/or any of their providers. A *voucher* is usually used to apply a rebate or a discount to the basket, but it itself does not have any form elements for content (texts, images et. alt.) that might be required for display once it is applied. Therefore it is dependent on such an uncategorized *default article* product that provides any desired information. Due to the nature of such a *voucher* – appling a rebate or discount – these non-categorized articles are sometimes also referred to as *rebate products* or *discount products*. 
```

Comment:
The original paragraph was better IMHO because the new one contains some false statements and mixes vouchers and discounts

Reply:
This is because you are fully aware with how things work. I on the other hand am not. 
What are "various discounts" of "various types"? What is so special about assigning the amount to the "configured (discount) article"? (I always assign an amount to an article, right?) How is the article "giving discount", when it does not appear anywhere in the frontend since it is not in a category? Where must it be assigned to "do its work"?
Please point out what is wring in my understanding, because I really thought that is how it works.